src/app/components/diary_list.jsx

```diff
@@ -0,0 +1,58 @@
+"use client"
+
+import React from 'react'
+import { useEffect, useState } from 'react'
+import { supabase } from '../lib/supabase';
+import Link from 'next/link';
+
+function DiaryList() {
+  const [entriesData, setEntriesData] = useState();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        let { data: diary_entries, error } = await supabase
+          .from('diary_entries')
+          .select('*')
+
+        if (error) {
+          throw error;
+        }
+        setEntriesData(diary_entries)
+      } catch (error) {
+      }
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <>
+      <ul role="list" className="divide-y divide-gray-100">
+        {entriesData ?
+          entriesData.map((entry) => (
+            <li key={entry.id} className="flex justify-between gap-x-6 w-screen max-w-7xl px-24 py-5">
+              <div className="flex min-w-0 gap-x-4">
+                <div className="min-w-0 flex-auto">
+                  <p className="text-sm font-semibold leading-6 text-gray-900">{new Date(entry.date).toLocaleDateString()}</p>
```

Comment:
would probably create a `formatDate` helper/util function so that all dates rendered would be identical. Not a biggie, moving on